openpype/hosts/max/plugins/load/load_camera_fbx.py

```diff
@@ -13,50 +22,76 @@ class FbxLoader(load.LoaderPlugin):
     order = -9
     icon = "code-fork"
     color = "white"
+    postfix = "param"
 
     def load(self, context, name=None, namespace=None, data=None):
         from pymxs import runtime as rt
-
         filepath = self.filepath_from_context(context)
         filepath = os.path.normpath(filepath)
         rt.FBXImporterSetParam("Animation", True)
         rt.FBXImporterSetParam("Camera", True)
         rt.FBXImporterSetParam("AxisConversionMethod", True)
+        rt.FBXImporterSetParam("Mode", rt.Name("create"))
         rt.FBXImporterSetParam("Preserveinstances", True)
         rt.ImportFile(
             filepath,
             rt.name("noPrompt"),
             using=rt.FBXIMP)
 
-        container = rt.GetNodeByName(f"{name}")
-        if not container:
-            container = rt.Container()
-            container.name = f"{name}"
+        namespace = unique_namespace(
+            name + "_",
+            suffix="_",
+        )
+        container = rt.container(
+            name=f"{namespace}:{name}_{self.postfix}")
+        selections = rt.GetCurrentSelection()
+        import_custom_attribute_data(container, selections)
 
-        for selection in rt.GetCurrentSelection():
+        for selection in selections:
             selection.Parent = container
+            selection.name = f"{namespace}:{selection.name}"
 
         return containerise(
-            name, [container], context, loader=self.__class__.__name__)
+            name, [container], context,
+            namespace, loader=self.__class__.__name__)
 
     def update(self, container, representation):
         from pymxs import runtime as rt
 
         path = get_representation_path(representation)
-        node = rt.GetNodeByName(container["instance_node"])
-        rt.Select(node.Children)
-        fbx_reimport_cmd = (
-            f"""
-
-FBXImporterSetParam "Animation" true
-FBXImporterSetParam "Cameras" true
-FBXImporterSetParam "AxisConversionMethod" true
-FbxExporterSetParam "UpAxis" "Y"
-FbxExporterSetParam "Preserveinstances" true
-
-importFile @"{path}" #noPrompt using:FBXIMP
-        """)
-        rt.Execute(fbx_reimport_cmd)
+        node_name = container["instance_node"]
+        node = rt.getNodeByName(node_name)
+        namespace, name = get_namespace(node_name)
+        sub_node_name = f"{namespace}:{name}_{self.postfix}"
+        inst_container = rt.getNodeByName(sub_node_name)
+        rt.Select(inst_container.Children)
+        transform_data = object_transform_set(inst_container.Children)
+        for prev_fbx_obj in rt.selection:
```

Comment:
Out of curiosity, what's the difference between `rt.selection` and `rt.GetCurrentSelection` ?

Reply:
> Out of curiosity, what's the difference between `rt.selection` and `rt.GetCurrentSelection` ?

Not much differences in this case.